.github/workflows/test-platforms.yaml

```diff
@@ -0,0 +1,126 @@
+name: Test Platforms
+
+on:
+    push:
+        branches:
+            - 3.x
+            - master
+    pull_request:
+
+env:
+    ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
Still needed?

Reply:
👍 I guess not (I don't know why was needed though).

Reply:
Can it be removed from dev kit then ? https://github.com/sonata-project/dev-kit/blob/master/templates/project/.github/workflows/test.yaml.twig#L15

Reply:
We need it if we set output

You can create a proof PR on admin bundle to check